Dockerfile

```diff
@@ -1,149 +1,186 @@
+                    -DEMBREE_ISPC_SUPPORT=OFF ; \
+        make 2>/dev/null -j${compile_cores} install ; \
+        rm -rf ${EMBREE_INSTALL_DIR}/build ${EMBREE_INSTALL_DIR}/embree ; \
+        # Clone and install MOAB
+        mkdir -p $HOME/MOAB && cd $HOME/MOAB ; \
+        git clone  --single-branch -b ${MOAB_BRANCH} --depth 1 ${MOAB_REPO} ; \
+        mkdir build && cd build ; \
+        cmake ../moab \
+                    -DENABLE_HDF5=ON \
+                    -DENABLE_NETCDF=ON \
+                    -DBUILD_SHARED_LIBS=OFF \
+                    -DENABLE_FORTRAN=OFF \
+                    -DENABLE_BLASLAPACK=OFF ; \
+        make 2>/dev/null -j${compile_cores} install ; \
+        cmake ../moab \
+                    -DENABLE_PYMOAB=ON \
+                    -DBUILD_SHARED_LIBS=ON \
+        make 2>/dev/null -j${compile_cores} install ; \
+        cd pymoab && bash install.sh; \
+        python setup.py install; \
+        rm -rf $HOME/MOAB ; \
+        # Clone and install Double-Down
+        mkdir -p $HOME/Double_down && cd $HOME/Double_down; \
+        git clone --single-branch -b ${DD_BRANCH} --depth 1 ${DD_REPO} ; \
+        mkdir build && cd build; \
+        cmake ../double-down \
+                    -DCMAKE_INSTALL_PREFIX=${DD_INSTALL_DIR} \
+                    -DMOAB_DIR=/usr/local \
+                    -DEMBREE_DIR=${EMBREE_INSTALL_DIR} ; \
+        make 2>/dev/null -j${compile_cores} install ; \
+        rm -rf ${DD_INSTALL_DIR}/build && ${DD_INSTALL_DIR}/double-down; \
+        # Clone and install DAGMC
+        mkdir -p $HOME/DAGMC && cd $HOME/DAGMC; \
+        git clone --single-branch -b ${DAGMC_TAG} --depth 1 ${DAGMC_REPO} ; \
+        mkdir build && cd build ; \
+        cmake ../DAGMC \
+                    -DBUILD_TALLY=ON \
+                    -DCMAKE_INSTALL_PREFIX=${DAGMC_INSTALL_DIR} \
+                    -DMOAB_DIR=/usr/local \
+                    -DDOUBLE_DOWN=ON \
+                    -DDOUBLE_DOWN_DIR=${DD_INSTALL_DIR} \
+                    -DCMAKE_PREFIX_PATH=${DD_INSTALL_DIR}/lib \
+                    -DBUILD_STATIC_LIBS=OFF \
+                    -DBUILD_STATIC_EXE=OFF ; \
+        make 2>/dev/null -j${compile_cores} install ; \
+        rm -rf ${DAGMC_INSTALL_DIR}/DAGMC ${DAGMC_INSTALL_DIR}/build ; \
+        # Clone and install OpenMC with DAGMC support
+        mkdir -p $HOME/OpenMC && cd $HOME/OpenMC; \
+        git clone --shallow-submodules --recurse-submodules -b ${OPENMC_BRANCH} --depth=1 ${OPENMC_REPO} ; \
+        mkdir build && cd build ; \
+        cmake ../openmc \
+                    -Doptimize=on \
+                    -Ddagmc=ON \
+                    -Ddebug=on \
+                    -DDAGMC_DIR=${DAGMC_INSTALL_DIR} \
+                    -DHDF5_PREFER_PARALLEL=on ; \
+        make 2>/dev/null -j${compile_cores} install ; \
+        cd ../openmc && pip install -e .[test] ; \
+    elif [ "$include_libmesh" = "true" ]; then \
```

Comment:
We can build OpenMC with DAGMC and libMesh support enabled. Is there a reason this isn't a separate `if` block?